lib/screens/home/widget/home_paging_widget.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:lydiaryanfluttersurvey/model/ui/survey_ui_model.dart';
+import 'package:lydiaryanfluttersurvey/screens/common/background_widget.dart';
+
+class HomePagingWidget extends StatelessWidget {
+  final List<SurveyUiModel> surveys;
+
+  const HomePagingWidget({super.key, required this.surveys});
+
+  @override
+  Widget build(BuildContext context) {
+    final PageController pageController = PageController();
+    return PageView(
+        controller: pageController,
+        children: surveys
+            .map((SurveyUiModel survey) => _buildSurveyItem(context, survey))
+            .toList());
+  }
+
+  Widget _buildSurveyItem(BuildContext context, SurveyUiModel survey) {
+    return Stack(
+      children: [
+        BackGroundWidget(
+          image: Image.network("${survey.coverImageUrl}l").image,
+        ),
+        SafeArea(
+          child: Column(
+            children: [
+              _buildSurveyHeader(context, survey),
+              const Spacer(),
+              _buildSurveyFooter(context, survey),
+            ],
+          ),
+        )
+      ],
+    );
+  }
+
+  Widget _buildSurveyHeader(BuildContext context, SurveyUiModel survey) {
+    return Container(
+      padding: const EdgeInsets.all(20),
+      child: Row(
+        children: [
+          Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                DateFormat("EEEE, MMMM dd")
+                    .format(survey.activeAt)
+                    .toUpperCase(),
+                style: Theme.of(context)
+                    .textTheme
+                    .titleLarge
+                    ?.copyWith(fontSize: 13),
+              ),
+              Text(
+                _daysSince(context, survey.activeAt),
+                style: Theme.of(context).textTheme.titleLarge,
+              )
+            ],
+            //TODO add user avatar
+          ),
+        ],
+      ),
+    );
+  }
+
+  String _daysSince(BuildContext context, DateTime dateTime) {
+    final dateFormat = DateFormat('yyyy-MM-dd');
+    final DateTime now = DateTime.parse(dateFormat.format(DateTime.now()));
+    final DateTime date = DateTime.parse(dateFormat.format(dateTime));
+    final difference = now.difference(date).inDays;
+    if (difference == 0) {
+      return AppLocalizations.of(context)!.today;
+    } else if (difference == 1) {
+      return AppLocalizations.of(context)!.yesterday;
+    } else {
+      return AppLocalizations.of(context)!.days_ago(difference);
+    }
+  }
+
+  Widget _buildSurveyFooter(BuildContext context, SurveyUiModel survey) {
```

Comment:
IMO, the texts should not be in the paging widget as they can be updated by the state depending on the list item. 🤔  So when we slide to change the page, it will stick at the foreground and just be updated. What do you think?